tests/test_meta/test_outlier_classifier.py

```diff
@@ -38,13 +41,52 @@ def test_obvious_usecase_quantile(dataset):
     clf_quantile.fit(X, y)
     assert clf_quantile.predict([[10, 10]]) == np.array([1])
     assert clf_quantile.predict([[0, 0]]) == np.array([0])
+    np.testing.assert_array_almost_equal(clf_quantile.predict_proba([[10, 10]]), np.array([[0, 1]]), decimal=4)
+    np.testing.assert_array_almost_equal(clf_quantile.predict_proba([[0, 0]]), np.array([[1, 0]]), decimal=4)
     assert isinstance(clf_quantile.score(X, y), float)
 
 
+def test_obvious_usecase_isolationforest(dataset):
+    X = dataset
+    y = (dataset.max(axis=1) > 3).astype(np.int)
+    outlier_model = IsolationForest(contamination=y.sum()/len(y))
+    clf_model = OutlierClassifier(outlier_model)
+    clf_model.fit(X, y)
+    assert clf_model.predict([[10, 10]]) == np.array([1])
+    assert clf_model.predict([[0, 0]]) == np.array([0])
+    np.testing.assert_array_almost_equal(clf_model.predict_proba([[0, 0]]), np.array([[1, 0]]), decimal=4)
+    outlier_proba = clf_model.predict_proba([[10, 10]])[0]
+    assert outlier_proba[0]<0.2
+    assert outlier_proba[1]>0.8
+    assert isinstance(clf_model.score(X, y), float)
+
+
+def test_obvious_usecase_oneclass_svm(dataset):
+    X = dataset
+    y = (dataset.max(axis=1) > 3).astype(np.int)
+    outlier_model = OneClassSVM(nu=0.05)
+    clf_model = OutlierClassifier(outlier_model)
+    clf_model.fit(X, y)
+    assert clf_model.predict([[10, 10]]) == np.array([1])
+    assert clf_model.predict([[0, 0]]) == np.array([0])
+    np.testing.assert_array_almost_equal(clf_model.predict_proba([[10, 10]]), np.array([[0, 1]]), decimal=3)
+    np.testing.assert_array_almost_equal(clf_model.predict_proba([[0, 0]]), np.array([[1, 0]]), decimal=3)
+    assert isinstance(clf_model.score(X, y), float)
```

Comment:
Could these two really not be combined via a `pytest.mark.parametrize` into a single test?

Reply:
I'm not familiar with `pytest.mark.parametrize` so I will look into it and implement it as you suggest

Reply:
Oh! It's amazing! 

You might appreciate this small guide: https://calmcode.io/pytest/clean-params.html

Reply:
I changed it such that pytest parametrization is used. Should I move `check_predict_proba` to `estimator_checks.py` and import it via `conftest.py`?

Reply:
thanks for your remarks below, this is indeed the way I should have used the `parametrize` function. I have adjusted in in the latest commit. Thanks for your help! 🙂